src/controllers/project.controller/updateProject/index.js

```diff
@@ -52,6 +52,15 @@ const updateProject = catchAsync(async (req, res) => {
     };
     await Hasura(updateProjectFlags, projectFlagsUpdateVariables);
 
+    const getTeamMembersResponse = await Hasura(getTeamMembers, { team_id: response.result.data.update_project.returning[0].team_id });
+    const teamMembers = getTeamMembersResponse.result.data.team_members;
+
+    const insertUserActivityResponses = await Promise.all(
```

Comment:
Please remove the await, just like we are not waiting while adding other activities.